CredentialProvider.Microsoft/Util/EnvUtil.cs

```diff
@@ -36,15 +36,19 @@ public static class EnvUtil
         public const string MsalFileCacheEnvVar = "NUGET_CREDENTIALPROVIDER_MSAL_FILECACHE_ENABLED";
         public const string MsalFileCacheLocationEnvVar = "NUGET_CREDENTIALPROVIDER_MSAL_FILECACHE_LOCATION";
 
-        private static readonly string LocalAppDataLocation = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData, Environment.SpecialFolderOption.Create), "MicrosoftCredentialProvider");
+        private static readonly string LocalAppDataLocation = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData, Environment.SpecialFolderOption.Create);
 
-        public static string AdalTokenCacheLocation { get; } = Path.Combine(LocalAppDataLocation, "ADALTokenCache.dat");
+        public static string AdalTokenCacheLocation { get; } = Path.Combine(LocalAppDataLocation, "MicrosoftCredentialProvider", "ADALTokenCache.dat");
```

Comment:
It feels like `"MicrosoftCredentialProvider"` should be a const, since it's shared b/w two paths.